dropwizard-jetty/src/test/java/io/dropwizard/jetty/HttpsConnectorFactoryTest.java

```diff
@@ -114,15 +114,33 @@ public void testExcludedProtocols() {
     }
 
     @Test
-    public void nonWindowsKeyStoreValidation() throws Exception {
+    void testDefaultExcludedProtocols() throws Exception {
+        HttpsConnectorFactory factory = new HttpsConnectorFactory();
+
+        SslContextFactory sslContextFactory = factory.configureSslContextFactory(new SslContextFactory.Server());
+        assertThat(sslContextFactory.getExcludeProtocols())
+                .containsExactlyElementsOf(factory.getExcludedProtocols());
+
+        sslContextFactory.start();
+        try {
+            assertThat(sslContextFactory.newSSLEngine().getEnabledProtocols())
+                    .doesNotContainAnyElementsOf(factory.getExcludedProtocols())
+                    .allSatisfy(protocol -> assertThat(protocol).doesNotStartWith("SSL"));
```

Comment:
Do we also want to ensure that enabled protocols only includes TLS v1.2/v1.3 and nothing else? Feels like a good test to fail if any older protocols are enabled.

Reply:
I also thought about adding this, but then the test will start failing when run on JVMs which support other (newer) protocols in the future.

Or maybe some JVM only supports up to TLSv2. 🤔